src/tracer.js

```diff
@@ -104,14 +104,14 @@ export default class Tracer {
         this._metrics.spansStarted.increment(1);
         if (span.context().isSampled()) {
             this._metrics.spansSampled.increment(1);
-            if (rpcServer) {
+            if (!firstInProcess) {
```

Comment:
wait, this is still wrong. If think we want
 * if parentId is not null
 ** trace started
 * else if rpcServer
 ** trace joined

firstInProcess can be both new and join